web/src/features/fba/components/map/FBAMap.tsx

```diff
@@ -33,6 +36,7 @@ import { buildHFICql } from 'features/fba/cqlBuilder'
 import { isUndefined, cloneDeep } from 'lodash'
 import LoadingBackdrop from 'features/hfiCalculator/components/LoadingBackdrop'
 import { Box } from '@mui/material'
+import { PMTILES_BUCKET } from 'utils/env'
```

Comment:
Minor: We can add this to the existing import of `utils/env` above

Reply:
Done